### PR TITLE
Adds git clone ros-planning/moveit2.git to install

### PR DESCRIPTION
**Pull request by newcanopies**
### Description

adds missing step to Foxy source install instructions: https://github.com/ros-planning/moveit2/issues/438#issuecomment-826421039 @tylerjw

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/getting_started/getting_started.rst
Comment (vatanaksoytezer) [inline doc/getting_started/getting_started.rst:35]: Formatting issue  in `cd ~/ws_moveit2/src  `: there is a trailing whitespace, which causes CI to fail (see https://github.com/ros-planning/moveit2_tutorials/actions/runs/785001092). Can you please remove it?

Also as a general comment please try to use pre-commit so you don't have formatting issues (see how to use pre-commit in the README https://github.com/ros-planning/moveit2_tutorials#formatting-and-style).

**Review by vatanaksoytezer — CHANGES_REQUESTED** [review]

**Comment by vatanaksoytezer** [comment]
@newcanopies Thanks for noticing the missing step and your PR! I left a small comment and will merge right after, you update your PR.

File: doc/getting_started/getting_started.rst
Comment (newcanopies) [reply doc/getting_started/getting_started.rst]: Spacing is removed. Thanks @vatanaksoytezer will use `pre-commit` next time.

**Comment by vatanaksoytezer** [comment]
Thanks for fixing it. @newcanopies you still have one more trailing whitespace in the empty line (see https://github.com/ros-planning/moveit2_tutorials/pull/79/checks?check_run_id=2437626373). Can you also remove that?

File: doc/getting_started/getting_started.rst
Comment (newcanopies) [reply doc/getting_started/getting_started.rst]: All space-trailing removed. 
(did not make a local clone to use `pre-commit` , hence easily missed when making quick browser edits).

**Review by vatanaksoytezer — APPROVED** [review]

**Pull request merged**
